src/test/java/io/kokuwa/maven/helm/junit/MojoExtension.java

```diff
@@ -113,7 +114,10 @@ public Object resolveParameter(ParameterContext parameterContext, ExtensionConte
 
 			// preconfigure
 
-			mojo.setChartDirectory(new File("src/test/resources/simple")); // set some sane defaults for tests
+			if (mojo instanceof AbstractChartDirectoryMojo) {
+				((AbstractChartDirectoryMojo) mojo)
+					.setChartDirectory(new File("src/test/resources/simple")); // set some sane defaults for tests
+			}
```

Comment:
Q: do we just ignore the chart directory if it's not an instance of AbstractChartDirectoryMojo?